src/Test/TestCombat.java

```diff
@@ -43,7 +46,8 @@ private boolean testUnitDeath()
     Unit infB = addUnit(testMap, testCo2, UnitEnum.INFANTRY, 1, 2);
 
     // Make sure the infantry will die with one attack
-    infB.damageHP(7);
+    CombatDamageModifier mod = new CombatDamageModifier(testCo1,500);
```

Comment:
It should realistically get its own test case, but I'm lazy and I didn't have stuff set up yet.